app/services/decisions/client_decision_tree.rb

```diff
@@ -183,6 +183,8 @@ def after_cannot_check_dwp_status
 
     def entered_appeal_reference_number?
       kase = current_crime_application.case
+      return false if kase&.case_type.nil?
```

Comment:
I'm not convinced we need the null safety check. Have you seen that we do, or just being cautious?

Reply:
Just being super cautious - I wrongly assumed there was no scenario in which case_type wouldn't be there, so don't want to make the same assumptions about case!